src/common/include/SIRF/common/sirf_image_data.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "data_container.h"
+
+/*
+\ingroup SIRF Image Data classes
```

Comment:
doxygen groups needs to be defined somewhere. i think needs to be a single word.